### ISSUE TITLE
[Feature] Проброс ссылок в навигационные компоненты.

### ISSUE THREAD

**Issue by zerodi** [zyfra/Prizm#295]
### Библиотека

- [ ] `@prizm-ui/core`
- [x] `@prizm-ui/components`
- [ ] `@prizm-ui/install`
- [ ] `@prizm-ui/icons`
- [ ] `@prizm-ui/flag-icons`
- [ ] `@prizm-ui/theme`
- [ ] `@prizm-ui/charts`
- [ ] `@prizm-ui/ast`
- [ ] `@prizm-ui/nx-plugin`
-

### Компонент

PrizmNavigationModule
PrizmNavigationMenuModule
PrizmTabsModule

### Предложение

Предлагаю реализовать в соответствующих компонентах возможность пробрасывать routerLink по умолчанию. Следовательно, если используется routerLink отображать элемент как активный при открытии страницы.

**Comment by ickisIckis** [comment]
Добрый день, уточните, пожалуйста момент по routerLink для табов. Что именно требуется?

**Comment by ickisIckis** [comment]
> Требуется добавить возможность использовать табы для семантически верной навигации по приложению (чтобы в табе была ссылка вида `<a href="...">`).

Для такого результата можно обернуть tabItem в ссылку.
примерно так:
```
  <prizm-tabs [(activeTabIndex)]="activeTabIndex">
    <a *ngFor="let item of tabs" [href]="item.link">
      <prizm-tab [content]="$any(item.title)" [type]="'line'"> </prizm-tab>
    </a>
  </prizm-tabs>
```
Навигацию взяли в работу.

**Comment by ickisIckis** [comment]
Компонент требует доработки описаний поведения для режима рубрикатора и папки, задача будет взята в работу, когда будут готовы эти два режима

**Comment by d-flasher** [comment]
Да, все перечисленные компоненты было бы удобно использовать вместе с angular routing, будь такая возможность.
